commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/conf/ConfigProperty.java

```diff
@@ -93,7 +93,9 @@
 
     DPU_UV_T_FILES_METADATA_POOL_PARTY_PASSWORD("dpu.uv-t-filesMetadata.pool.party.password"),
 
-    USE_LOCALIZED_DPU_NAME("dpu.name.localized");
+    USE_LOCALIZED_DPU_NAME("dpu.name.localized"),
+
+    FRONTEND_INITIAL_PAGE("frontend.initial.page");
```

Comment:
Please adjust frontend config.properties.sample, so that it contains sample configuration of "frontend.initial.page", commented out.